tests/integration/test_multi_stem_verification.py

```
@@ -156,27 +156,53 @@ def test_four_stem_separation(input_file, ref_prefix, tmp_path):
+    Uses drumkit_groove.flac (a real drum performance) fed directly into
+    DrumSep, so every kit part has audible content — unlike the previous
+    levee_drums clip, whose drums lacked ride content, leaving the ride stem
+    silent and its correlation check meaningless. htdemucs drum extraction is
+    covered separately by test_four_stem_separation.
+    """
+    input_file = "tests/inputs/drumkit_groove.flac"
+    output_dir = str(tmp_path)
 
-    print(f"  Step 2: Split drums into kit parts using DrumSep")
-    step2_files = run_separation("MDX23C-DrumSep-aufr33-jarredou.ckpt", drums_file, output_dir)
+    print(f"\n  Split drum kit into kit parts using DrumSep")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)